woocommerce/payment-gateway/payment-tokens/class-sv-wc-payment-gateway-payment-tokens-handler.php

```diff
@@ -590,6 +585,37 @@ public function get_tokens( $user_id, $args = array() ) {
 	}
 
 
+	/**
+	 * Gets token objects from the legacy user meta data store.
+	 *
+	 * @since x.y.z
+	 *
+	 * @param int $user_id WordPress user ID
+	 * @param null|string $environment_id desired environment ID
+	 * @return SV_WC_Payment_Gateway_Payment_Token[]
+	 */
+	public function get_legacy_tokens( $user_id, $environment_id = null ) {
+
+		if ( null === $environment_id ) {
+			$environment_id = $this->get_environment_id();
+		}
+
+		$this->legacy_tokens[ $environment_id ][ $user_id ] = [];
```

Comment:
Based on the way `get_tokens()` works, I think we want to check whether we already have tokens on the cache and return those instead before this line.